lib/config.js

```diff
@@ -333,7 +333,11 @@ function Config(options) {
 
     if (useConfig) {
         debug("Using command line config " + useConfig);
-        this.useSpecificConfig = loadConfig(path.resolve(process.cwd(), useConfig));
+        try {
+            this.useSpecificConfig = loadConfig(path.resolve(process.cwd(), useConfig));
```

Comment:
Yes, I tried to avoid try-catch but currently `--config` option accepts a relative path whose first letter would be alphanumeric.